src/provider.rs

```diff
@@ -171,14 +229,43 @@ impl Builder {
         auth_token: AuthToken,
         events: broadcast::Sender<Event>,
         cancel_token: CancellationToken,
+        rpc: E,
+        internal_rpc: impl ServiceEndpoint<ProviderService>,
```

Comment:
I probably missed this, but why do we need `rpc` and `internal_rpc`?

Reply:
We don't strictly speaking need it. There is a feature in quic-rpc that allows you to combine two rpc mechanisms. E.g. in memory for internal use and quinn for inter process.

But I tried it out, and it leads to very long types (quic-rpc currently does not support boxing, so you have to drag the types around). Also, you want to process errors for internal rpc calls differently than errors from external calls. An accept error in the internal rpc means that the controller has been dropped, which means that you probably want to exit. An error in an accept from an external source is something you just want to log and otherwise ignore.

Reply:
> (for some reason this did not make it to docs.rs because it is disabled by default... How do I change that?)

`#[cfg(any(feature = "the_feature", doc))]` if my memory is right